src/compiler/types.ts

```diff
@@ -415,7 +415,7 @@ namespace ts {
         BlockScoped = Let | Const,
 
         ReachabilityCheckFlags = HasImplicitReturn | HasExplicitReturn,
-        EmitHelperFlags = HasClassExtends | HasDecorators | HasParamDecorators | HasAsyncFunctions,
+        EmitHelperFlags = HasClassExtends | HasDecorators | HasParamDecorators | HasAsyncFunctions | HasJsxSpreadAttribute,

```

Comment:
side note, we should get rid of this flag, specially that it is not used except in one place, and is at bit 30, which will cause a conversion on node-32 bit.